### DISCUSSION TITLE
Refactor ButtonGroup stories to reduce code repetition by unifying shared options

### DISCUSSION THREAD

**Discussion by VeerChaurasia** [mondaycom/vibe#2538] [Ideas]
In the file of:
`packages/core/src/components/ButtonGroup/__stories__/ButtonGroup.stories.tsx`
ButtonGroup stories currently repeat the same set of button options (Alpha, Beta, Gamma, Delta) across multiple stories. To improve maintainability and reduce redundancy, we should refactor the code to use a shared variable for these options.

**Comment by YossiSaadi** [comment]
Thanks for suggesting this @VeerChaurasia!
We're looking to make our stories very explicit
If we use vars and helpers to store stuff (which is done already on a lot of stories), the one checks the "Show code" section in Storybook or "Edit Story" section, will see code that is unreachable for him.

E.g if we store ll the button options as a `const options = [ ... ]` and we'll use it in stories, people will see word `options` and have no idea how they should implement - what should `options` be - an array? an object? what is the type of each object?
